docs/submission/validating-molecular-data.md

```diff
@@ -411,29 +414,55 @@ seq-tools-in-docker validate -d submission-directory/*/*.json
 <!---  Tabs end here -->
 
 
-You will see an interactive prompt of the results of validation as the tool runs.  Once completed, view the details of the validation report:
+You will see an interactive prompt of the results of validation of each metadata payload as the tool runs.  Once completed, view the details of the validation reports under the current folder:
 
 ```bash
 cat validation_report.PASS.jsonl | jq . | less
 cat validation_report.INVALID.jsonl | jq . | less
 ```
 ### Scriptable Location Validation
 
-If you do not want to use the structured directory, you can use the '-d' option if data files are located in a different directory than where the metadata payloads are listed.
+If your data files are located in a different directory than where the metadata payloads are listed, and you do NOT want to re-organize them into the above structured directory, you can use the '-d' option to provide the data files location. You can also use this to validate all metadata payloads in a directory at once using a wildcard.  
```

Comment:
is it `reorganize`? 
When I look into google further seems these words can be hyphenated..so up to you:)

Reply:
https://www.thepunctuationguide.com/apostrophe.html#:~:text=The%20general%20rule%20is%20that,ends%20in%20s%20or%20not.&text=The%20possessive%20of%20a%20plural,a%20letter%20other%20than%20s. 

`The apostrophe is seldom used to form a plural noun.`

As there is no real need to add possession here, i think that a plains with no apostorphe is most clear.  i will remove your instead.

Reply:
but the location belongs to the data files?

Reply:
I always learned this..but maybe it's changed

The possessive of a plural noun is formed by **adding only an apostrophe when the noun ends in s**, and by adding both an apostrophe and s when it ends in a letter other than s.